src/components/StickerTemplate.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {
+    View,
+    Text,
+    Image,
+    StyleSheet,
+    ImageBackground,
+    Dimensions,
+} from 'react-native';
+// Imagenes de Plantilla
+import fondo from '../../assets/appAssets/template/fondo.jpg';
+import marco from '../../assets/appAssets/template/marco.png';
+import delantero from '../../assets/appAssets/test/delantero.png';
+import defensa from '../../assets/appAssets/test/defensa.png';
+import medioCentro from '../../assets/appAssets/test/medioCentro.png';
+import arquero from '../../assets/appAssets/test/arquero.png';
+
+const { width } = Dimensions.get('window');
+
+// Mientras no se tenga banderas
+import bandera from '../../assets/appAssets/test/bandera.png';
+
+
+export default function StickerTemplate({ sticker }) {
+    return (
+        <View style={styles.container}>
+            <ImageBackground
+                source={fondo}
+                style={styles.marco}
+            >
+                <View style={{ alignItems: 'center' }}>
+                    <Image
+                        source={{uri:sticker.img}}
+                        style={[styles.jugador, {top: 8}]}
+                    />
+                </View>
+                <View style={{ position: 'absolute' }}>
+                    <Image
+                        source={{uri:sticker.team.badge}}
+                        style={[styles.bandera, { left: width / 4.5 }]}
+                    />
+                    <Image
+                        source={
+                            (sticker.position=='Arquero')?(arquero):
+                            ((sticker.position=='Defensa')?(defensa):
+                            ((sticker.position=='MedioCentro')?(medioCentro):
+                            ((sticker.position=='Delantero')?(delantero):(delantero))
+                            ))
+                        }
```

Comment:
Haz un hook personalizado para esto, de modo que quede:

```suggestion
                        source={playerRoleImg}
```